### PR TITLE
added python3 support to simulated dataset

### PR DESCRIPTION
**Pull request by avinashbarnwal**
The previous code is written for python2. Now, python2  is obsolete. Please find the python3 support.

### CODE REVIEW

File: simulated.datasets.py [hunk 1]
```diff
@@ -1,7 +1,8 @@
-import matplotlib.pyplot as plt
+from   matplotlib import pyplot as plt
```
Comment (aldro61) [inline simulated.datasets.py]: Remove extra space after from

File: simulated.datasets.py [hunk 2]
```diff
@@ -1,7 +1,8 @@
...
 import numpy as np
 import os
-import seaborn as sns; sns.set_style("white")
+import seaborn as sns;
```
Comment (aldro61) [inline simulated.datasets.py]: Remove `;` now that set_style is on another line.

File: simulated.datasets.py [hunk 3]
```diff
@@ -70,12 +73,16 @@ def save_dataset(X, y, plot, n_folds, name):
         folds = np.arange(X.shape[0]) % n_folds + 1
         random_state.shuffle(folds)
 
+        paren_dir = "data"
```
Comment (aldro61) [inline simulated.datasets.py]: Typo: parent

File: simulated.datasets.py [hunk 4]
```diff
@@ -102,4 +109,4 @@ def save_dataset(X, y, plot, n_folds, name):
     datasets = {"simulated.sin": lambda x: np.sin(x),
                 "simulated.abs": lambda x: np.abs(x - 5.),
                 "simulated.linear": lambda x: x / 5}
-    generate_function_datasets(datasets, random_seed=4)
+    generate_function_datasets(datasets, random_seed=4)
```
Comment (aldro61) [inline simulated.datasets.py]: Add empty line at end of file.

**Review by aldro61 — CHANGES_REQUESTED** [review]
Thanks for adapting the code to Python 3! I requested a few very minor changes. Could you please address them and then I'll merge.

**Review by avinashbarnwal — COMMENTED** [review]
Please find the changes required by you.

File: simulated.datasets.py (hunk 1 above)
Comment (avinashbarnwal) [reply simulated.datasets.py]: Done

File: simulated.datasets.py (hunk 2 above)
Comment (avinashbarnwal) [reply simulated.datasets.py]: Done

File: simulated.datasets.py (hunk 3 above)
Comment (avinashbarnwal) [reply simulated.datasets.py]: Done

File: simulated.datasets.py (hunk 4 above)
Comment (avinashbarnwal) [reply simulated.datasets.py]: Done

**Comment by aldro61** [comment]
Looks good to me. Thanks for your contribution!

**Review by aldro61 — APPROVED** [review]

**Pull request merged**